### ISSUE TITLE
Insufficient contrast ratio on Quick Info text color in High Contrast #2

### ISSUE THREAD

**Issue by dpoeschl** [dotnet/roslyn#10099]
![image](https://cloud.githubusercontent.com/assets/235241/14046775/b6aa25ee-f260-11e5-927c-62e755470763.png)

**Comment by dpoeschl** [comment]
I guess we need to detect that we're in a High Contrast theme and then use the appropriate system color instead of trying to use the classified colors (which is where the green text is coming from).

**Comment by Pilchie** [comment]
Moving to 2.1 as we already shipped VS 2015 this way.

**Comment by dpoeschl** [comment]
The background color for tooltips seems to be the real problem here, and it looks like it's been fixed:

![image](https://cloud.githubusercontent.com/assets/235241/26503218/a8bea858-41f4-11e7-8a5e-edd3ea2deb02.png)
